besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -817,6 +817,13 @@ void setBannedNodeIds(final List<String> values) {
       arity = "1")
   private final Wei txFeeCap = DEFAULT_RPC_TX_FEE_CAP;
 
+  @Option(
+      names = {"--rpc-allow-unprotected-txs"},
+      description =
+          "Allow for unprotected (non EIP155 signed) transactions to be submitted via RPC (default: ${DEFAULT-VALUE})",
+      arity = "1")
+  private final Boolean unprotectedTransactionsAllowed = false;
```

Comment:
This breaks backwards compatibility, so it will have to wait for the next quarterly release of the default is to deny.
I would recommend adding the flag with the default to allow and then at the next quarterly release rc cycle flip the flag to deny.

Reply:
Makes sense.

Reply:
done